src/main/java/com/minecolonies/coremod/client/render/mobs/barbarians/AbstractRendererBarbarian.java

```diff
@@ -0,0 +1,55 @@
+package com.minecolonies.coremod.client.render.mobs.barbarians;
+
+import com.minecolonies.coremod.entity.ai.mobs.barbarians.AbstractEntityBarbarian;
+import com.minecolonies.coremod.entity.ai.mobs.barbarians.EntityBarbarian;
+import com.minecolonies.coremod.entity.ai.mobs.barbarians.EntityChiefBarbarian;
+import net.minecraft.client.model.ModelBiped;
+import net.minecraft.client.renderer.entity.RenderLiving;
+import net.minecraft.client.renderer.entity.RenderManager;
+import net.minecraft.client.renderer.entity.layers.LayerBipedArmor;
+import net.minecraft.client.renderer.entity.layers.LayerHeldItem;
+import net.minecraft.util.ResourceLocation;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Abstract for rendering Barbarians.
+ */
+public class AbstractRendererBarbarian extends RenderLiving<AbstractEntityBarbarian>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this class abstract or rename it, since it matches the regular expression '^Abstract[A-Z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00118)